src/App.tsx

```diff
@@ -17,18 +82,21 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onSearch={setSearch}
+                onStatusChange={setStatus}
+                onClearSearch={() => setSearch('')}
+              />
```

Comment:
Verify that the `TodoFilter` component correctly handles the props and updates the state as expected. This is important for filtering functionality.